include/swift/SIL/SILInstruction.h

```diff
@@ -5234,27 +5276,45 @@ class SelectEnumInstBase
   /// not to need this.
   NullablePtr<EnumElementDecl> getSingleTrueElement() const;
 };
-  
+
+/// A select enum inst that produces a static OwnershipKind set upon the
```

Comment:
I don't know what makes the Ownership kind "static" since it appears to be dynamically selected based on the operands.